### DISCUSSION TITLE
PORNDB can't find scene

### DISCUSSION THREAD

**Discussion by hosseinc** [stashapp/stash#5557] [Q&A] [answered]
hi

when i use scrap with porndb in edit tab of a specefic scene, it says no scene found but however i have found that scene in thepo$r@ndb.net databse. is there a way to manually add?

i have tried porndb url of the scene in the url box and also tried putting endpoint there and did'nt get any results.

**Comment by DogmaDragon** [comment]
You can copy UUID (known as StashID in Stash) of the scene into search or query field to get that exact scene.

**Comment by Maista6969** [comment, marked as answer]
Unfortunately this only works with stash-boxes, since TPDB have their own implementation you'd have to ask them to implement support for this in their search
